.github/workflows/cd.yml

```diff
@@ -30,17 +30,20 @@ jobs:
       - name: Install build dependency
         run: python3 -m pip install --constraint requirements/build.txt build
 
-      - name: Build binary wheel and source tarball
-        run: PIP_CONSTRAINT=requirements/build.txt python3 -m build --sdist --wheel --outdir dist/ .
+      - name: Build binary wheel, source tarball and changelog
+        env:
+          TAG: ${{ github.ref_name }}
```

Comment:
I suppose this could be removed since `GITHUB_REF_NAME` should be in the environment already